CHANGELOG.md

```diff
@@ -24,6 +24,10 @@
   [Ben Yohay](https://github.com/byohay)
   [#861](https://github.com/CocoaPods/Xcodeproj/pull/861)
 
+* Add `code_coverage_targets` support for `TestAction`.
```

Comment:
nit, if possible please add two empty spsaces after this as it helps with markdown formatting.